brave/brave-core
# fix: password-protect account private key reveal

Resolves https://github.com/brave/brave-browser/issues/24322

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Unlock the Brave wallet
2. Navigate to Accounts -> Select Account1 -> Account details -> Private Key
3. "Show key" button should be disabled & and password field should be visible
4. Enter wallet password into the password field
5. "Show key" button should now be enabled
6a. Error message appears if incorrect password was entered
6b. Key is displayed if correct password was entered

![Screen Shot 2022-07-27 at 6 08 02 PM](https://user-images.githubusercontent.com/30185185/181650435-29279049-07f1-41be-8afc-1ad155226fe8.png)
![Screen Shot 2022-07-27 at 6 08 13 PM](https://user-images.githubusercontent.com/30185185/181650438-993137cd-28bf-4c21-a98b-535fc73463d6.png)
![Screen Shot 2022-07-27 at 6 09 59 PM](https://user-images.githubusercontent.com/30185185/181650442-54765c73-9b00-4bec-a48a-ca04d95cca48.png)
![Screen Shot 2022-07-27 at 6 12 50 PM](https://user-images.githubusercontent.com/30185185/181650445-945ea458-1009-450f-9139-fca2b7656b26.png)
